lib/screen/login/login_page.dart

```diff
@@ -57,7 +67,7 @@ class _LoginPageState extends State<LoginPage> {
                         boxShadow: [
                           BoxShadow(
                               color: const Color(0xcbf4f4f4),
-                              offset: Offset(-5, -5),
+                              // offset: Offset(-5, -5),
```

Comment:
@bstrumillo flutter doesn't like negative offsets for some reason. I was getting a weird output thrown in the debug window